divergent-studio/src/components/Banner/styles.css

```diff
@@ -1,33 +1,101 @@
-.header {
+.icon {
   position: fixed;
-  margin-right: 65px;
-  margin-top: 29px;
+  left: 100px;
+  top: 30px;
+  text-align: right;
 }
-
-.container {
-  background-color: blue;
-}
-
-button {
-  width: 202px;
+.contact {
+  position: fixed;
+  right: 100px;
+  top: 30px;
+  text-align: right;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
   height: 53px;
+  width: 220px;
+  padding: 0 25px;
+  background-color: #000000;
+  border: none;
+  color: white;
+  text-align: center;
+  text-decoration: none;
   filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
 }
 
-.banner-container {
+.contact:hover {
+  text-decoration: underline;
+}
+
+.container {
   width: 100vw;
   height: 100vh;
+  background-image: url(../../assets/background.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
   display: flex;
-  justify-content: right;
   align-items: center;
+  justify-content: right;
+}
+
+.text {
+  width: 50%;
```

Comment:
Maybe here we need some mediaQuery to avoid being 50% in small screens. Maybe we can use max-width